src/scss/skin-modern/_skin.scss

```diff
@@ -51,7 +51,7 @@
   @import 'skin-smallscreen';
   @import 'skin-tv';
 
-  .#{$prefix}-ui-uicontainer {
+  &.#{$prefix}-ui-uicontainer {
```

Comment:
This went unnoticed as some of the general styles are handled by the player itself.